### PR TITLE
Add cardList

### PR DESCRIPTION
**Pull request by kepet19**
- [x] cardList should have a filter object. that can be update from parrent. 
- [x] cardList should have a `useEffect` with a dependency on `filter object` so every time `filter object` updates it update the `CardList`
- [x] search.js should have a filter object it can update and pass it down to cardList.

### CODE REVIEW

File: components/views/save.js [hunk 1]
```diff
@@ -1,108 +1,18 @@
...
-      <TouchableOpacity
-        style={saveStyle.remove}
-        onPress={() => {
-          Database.removeSaved(item.id, username);
-        }}
-      />
-    </View>
-    <View style={saveStyle.cardContent}>
-      <Text style={saveStyle.name}>{item.name}</Text>
+export default function SearchScreen({ navigation }) {
```
Comment (Joergenson) [inline components/views/save.js:5]: Cool but in saved all the cards just need to have "star" as icon. It doesn't need the start outline part.

**Comment by Joergenson** [comment]
Also I get merge conflicts when pulling

**Comment by Joergenson** [comment]
> Also I get merge conflicts when pulling

It is saying I have local commits to push in your name 🧇

File: components/views/save.js (hunk 1 above)
Comment (Joergenson) [reply components/views/save.js:5]: But then again. since you filter then i guess they just have a filled star

Comment (kepet19) [reply components/views/save.js:5]: Not sure how to change that :b

**Comment by kepet19** [comment]
> > Also I get merge conflicts when pulling
> 
> It is saying I have local commits to push in your name waffle

yeah it is because of rebase, Whoops :b 
you can either delete the branch or git reset origin/main. 
you can't right now, I will have to solve the conflict again with main :b

**Comment by Joergenson** [comment]
> > > Also I get merge conflicts when pulling
> > 
> > 
> > It is saying I have local commits to push in your name waffle
> 
> yeah it is because of rebase, Whoops :b you can either delete the branch or git reset origin/main. you can't right now, I will have to solve the conflict again with main :b

Nice works now. Give me 5 min to fix issues

**Comment by Joergenson** [comment]
closes #84

**Review by Joergenson — APPROVED** [review]
Nice!

**Pull request merged**